vivarium/processes/growth_rate.py

```diff
@@ -22,11 +22,22 @@ class GrowthRate(Process):
     """ A Vivarium process that models exponential growth of biomass """
 
     name = NAME
-    defaults = {
-        'default_growth_rate': 0.0005,
-        'default_growth_noise': 0.0,
-        'variables': ['mass']
-    }
+    # defaults = {
+    #     'default_growth_rate': 0.0005,
+    #     'default_growth_noise': 0.0,
+    #     'variables': ['mass']
+    # }
+
+    def __init__(
+            self,
+            default_growth_rate=0.0005,
+            default_growth_noise=0.0,
+            variables=('mass',),
+            **base_parameters):
+
+        parameters = locals().copy()
+        base_parameters = parameters.pop('base_parameters')
+        self.initialize(parameters, base_parameters)
```

Comment:
I know I helped create this thing, but on reflection I feel the cost here for `Process` creators to have to add this four-line litany to every `__init__` method is too high. I appreciate what we're trying to solve here but the trade-off can't be to create cryptic things we demand of our users.... IMO

Reply:
Maybe we can make this optional? If people want function keywords, they can follow this template. But the previous `__init__` approach with defaults and a parameters dict can work just as well.